runtime/server/x86/bin/BUILD

```diff
@@ -1,14 +1,13 @@
 
-
 cc_binary(
     name = "decoder_main",
     srcs = ["decoder_main.cc"],
     deps = [
         "@com_github_google_glog//:glog",
         "@com_github_gflags_gflags//:gflags",
         "@libtorch//:torch",
-        "//frontend:frontend",
         "//decoder:decoder",
+        "//frontend:frontend",
```

Comment:
Please use tools for auto-formatting.

Reply:
I didn't make the tool work, maybe refine it in the future

Reply:
I read about this https://docs.bazel.build/versions/master/skylark/build-style.html, and refine it by hand.